TestCoreApp_Elliott/Areas/Olympics/Views/Olympics/Index.cshtml

```diff
@@ -1,4 +1,6 @@
 ﻿@using TestCoreApp_Elliott.Models.OlympicGames
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
We need this line so that the 'asp-*` tags work. Typically, this will be added in the `Views\_ViewImports.cshtml` but you don't have that file in the Olympics area.